RecoBTag/ONNXRuntime/interface/tensor_fillers.h

```diff
@@ -18,6 +19,30 @@ namespace btagbtvdeep {
   void seedTrack_tensor_filler(float*& ptr, const btagbtvdeep::SeedingTrackFeatures& seed_features);
 
   void neighbourTrack_tensor_filler(float*& ptr, const btagbtvdeep::TrackPairFeatures& neighbourTrack_features);
+ 
+  std::vector<float> inputs_parT(const btagbtvdeep::ChargedCandidateFeatures& c_pf_features, parT::InputIndexes idx);
+
+  std::vector<float> inputs_parT(const btagbtvdeep::NeutralCandidateFeatures& n_pf_features, parT::InputIndexes idx);
+
+  std::vector<float> inputs_parT(const btagbtvdeep::SecondaryVertexFeatures& sv_features, parT::InputIndexes idx);
+
+  template<class parT_features>
+  void parT_tensor_filler(float*& ptr, parT::InputIndexes idx , const parT_features pf) {
```

Comment:
pass complex objects by reference: `const parT_features& pf`

Reply:
agree, code is modified